README.md

```diff
@@ -1,25 +1,26 @@
-# public-upload-to-imgur
-A GitHub Action that can upload "anonymous" images to Imgur using their API and give you the URL of the newly created image.
-Note: "anonymous" means that the image is not tied to an account (see [this](https://apidocs.imgur.com/#intro)).
+# Publish to Imgur
+
+A GitHub Action that can upload "anonymous" images to Imgur using their API that gives you the URL of the newly created image.
+Note: By "anonymous" we mean that the image is not tied to an account (see [this](https://apidocs.imgur.com/#intro)).
 
 **Purpose** 
 
-This action is used as part of the peek-icons workflow in the [devicon](https://github.com/devicons/devicon) repo to automate the
-icon checking process. Our workflow would generate screenshots using Selenium, upload it to Imgur (using this action) then publish a comment on a PR showing the result (using [another action](https://github.com/marketplace/actions/pr-comment)).
+This action is used as part of the Peek Icons workflow in the [Devicon](https://github.com/devicons/devicon) repository to automate the
```

Comment:
I liked "peek-icons" and "devicon" more, but I don't think there is a official way of writing "devicon/Devicon".

Reply:
> I liked "peek-icons" and "devicon" more

@Thomas-Boi, thoughts?

Reply:
I prefer the old way more since that's how we named them for the labels and our organization. I'd say keep the old spelling but I'm open to "Devicon".

This could be something to discuss. [Google](https://github.com/orgs/google/repositories) has a proper capital case but we don't => is our official name `devicon` then?

Reply:
So, what should we use then: `Devicon` or `devicon`?

Reply:
Let's go with Devicon. That's what we have on the website:

![image](https://user-images.githubusercontent.com/43018778/129783002-526a802c-6a26-42e9-ae20-33bf24eb0761.png)